src/net/README.md

```diff
@@ -25,43 +25,35 @@ The setup we start with and later move `eth4` from `lag0` to `br0`.
                  eth4  eth5
 
 Consider the case when `next` contains `0`, net reads all interfaces, in
-dependency order, from `/run/net/0/` running all `ip-link` and `ip-addr`
-commands.  When done, it writes `0` to the file `/run/net/gen` and then
-removes the `next` file to confirm the generation has been activated.
+dependency order, from `/run/net/0/` running all `cmd.init` scripts.
```

Comment:
Just a thought about the `cmd.init` vs `init.cmd` dilemma:

Since we're feeding `ip.init` to `/sbin/ip` - does it not make more sense to have the file extension be `.ip` (since that is the "format" of the file)?

Perhaps helpful: if we add shell scripts, do we want to call them `sh.init` or `init.sh`?

Reply:
Yes, fully agree, I'll rename and push a fix!